### DISCUSSION TITLE
Use UnifiedPush when FCM isn't available

### DISCUSSION THREAD

**Discussion by d4f5409d** [community/community#73167] [Mobile]
When we have no Google Mobile Services installed, we could still get GitHub notifications with UnifiedPush

**Comment by elisealix22** [comment]
Hi @d4f5409d , thanks for recommending this! We will take a look internally on our Android team. Let us know if you have any other recommendations or feature requests in the meantime.

**Reply by d4f5409d** [reply]
Thank you :)
